package.json

```diff
@@ -41,24 +41,23 @@
     "@carbon/layout": "^10.37.1",
     "@carbon/type": "^10.44.0",
     "carbon-web-components": "^1.21.0",
-    "lit": "^2.2.4",
```

Comment:
Please, don't change the `package.json` & `package-lock.json`.

Reply:
Done!
Sorry, they were added when I brought the changes from main.